tests/phpunit/CRM/Financial/BAO/PaymentProcessorTypeTest.php

```diff
@@ -43,6 +43,7 @@ public function testCreate() {
       'name' => 'Test_Payment_Processor',
       'title' => 'Test Payment Processor',
       'billing_mode' => 1,
+      'class_name' => 'Payment_Dummy',
```

Comment:
note that becaue there is no default value for class_name yet i am setting it as default is the reason for the changes here